test/src/test/java/org/corfudb/integration/CorfuReplicationLargeTxIT.java

```diff
@@ -62,10 +69,9 @@ public void testAtomicSnapshotSyncWithoutChunking() throws Exception {
     }
 
     /**
-     * With the max write size of EIGHT_THOUSAND, it was empirically
-     * determined that 20 entries in a table had a serialized size of 7.5K
-     * approx.  Hence, a snapshot sync with twice the data(40 entries) will be
-     * applied in 2 transactions.
+     * MAX_WRITE_SIZE_BYTES is the maximum number of bytes which can be written in a single transaction during
```

Comment:
This comment is duplicated, it should be enough if placed along the variable definition.